dipy/reconst/csdeconv.py

```diff
@@ -118,18 +120,20 @@ def __init__(self, gtab, response, reg_sphere=None, sh_order=8, lambda_=1,
         # scale lambda_ to account for differences in the number of
         # SH coefficients and number of mapped directions
         # This is exactly what is done in [4]_
-        self.lambda_ = (lambda_  * self.R.shape[0] * r_rh[0] /
-                        (np.sqrt(self.B_reg.shape[0]) * np.sqrt(362.))
-                       )
+
+        lambda_ = (lambda_  * self.R.shape[0] * r_rh[0] /
+                   (np.sqrt(self.B_reg.shape[0]) * np.sqrt(362.)))
```

Comment:
On Tue, Jan 27, 2015 at 1:21 PM, Samuel St-Jean <notifications@github.com>
wrote:
>
> Well, if I change the regularization sphere, without changing lambda, that
> would defeat the purpose of changing the regularization sphere if $N_0$ is
> set to 362, and not the actual sphere I passed. And existing code would
> only be different if someone is passing another sphere in this case, where
> it would probably make more sense to use something else than 362. Or I
> don't see why the arbitrary $\lambda' ~ 20 * \lambda$ and it's just a quirk
> that sqrt(362) ~ 20.
>
>
Can you please explain this? I don't understand.